src/sass/section4.scss

```diff
@@ -0,0 +1,121 @@
+/* Section 4 / Top Sellers Section */
+
+.top-sellers-title {
+  margin-bottom: 32px;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 1.16;
+  text-align: center;
+  letter-spacing: -0.03em;
+  text-transform: uppercase;
+}
+
+.top-sellers-title span {
+  color: #fd9222;
+}
+
+.sellers-items {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 32px 16px;
+  margin-bottom: 20px;
+}
+
+.sellers-item {
+  max-width: 335px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.sellers-item:last-child {
+  display: none;
+}
+
+.top-sellers-text-box {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+.top-sellers-text {
+  margin-top: 20px;
+  max-width: 306px;
+}
+
+.tablet-text {
+  display: none;
+}
+
+.top-sellers-buy-now-btn {
+  width: 120px;
+  height: 41px;
+  padding: 12px, 28px, 12px, 28px;
+  border-radius: 100px;
+  gap: 14px;
+  color: var(--white-color);
+  background-color: rgba(253, 146, 34, 1);
+  border: transparent;
+}
+
+/* Tablet */
+@media screen and (min-width: 767px) {
+  .container {
+    max-width: 744px;
```

Comment:
in figma e 704px

Reply:
ori ii dau padding 0 cu max-width: 704px; la container, ori max-width: 744px. altfel nu intra ul in el ca sa afiseze corect li-urile